docs/python/managing-python-environments-in-visual-studio.md

```diff
@@ -256,15 +256,16 @@ To correct an environment you wish to keep, first try using its installer's **Re
 To correct an environment that doesn't have a repair option, or to remove an invalid environment, use the following steps to modify the registry directly. Visual Studio automatically updates the **Python Environments** window when you make changes to the registry.
 
 1. Run *regedit.exe*.
-1. Navigate to **HKEY_LOCAL_MACHINE\SOFTWARE\Python**. For IronPython, look for **IronPython** instead.
+1. Navigate to **HKEY_LOCAL_MACHINE\SOFTWARE\Python** or **HKEY_CURRENT_USER\SOFTWARE\Python**. For IronPython, look for **IronPython** instead.
 1. Expand the node that matches the distribution, such as **Python Core** for CPython or **ContinuumAnalytics** for Anaconda. For IronPython, expand the version number node.
 1. Inspect the values under the **InstallPath** node:
 
     ![Registry entries for a typical CPython installation](media/environments/environments-registry-entries.png)
 
     - If the environment still exists on your computer, change the value of **ExecutablePath** to the correct location. Also correct the **(Default)** and **WindowedExecutablePath** values as necessary.
     - If the environment no longer exists on your computer and you want to remove it from the **Python Environments** window, delete the parent node of **InstallPath**, such as **3.6** in the image above.
-
+    - Invalid settings in **HKEY_CURRENT_USER\SOFTWARE\Python** overide the settings in **HKEY_LOCAL_MACHINE\SOFTWARE\Python**
```

Comment:
typo, should be 'override'

Reply:
Fixed in latest commit.

Reply:
Thanks @huguesv for taking a look and @pcartwright81 for the PR.